### DISCUSSION TITLE
[GSoC] Introduction and Discussion of Potential Project

### DISCUSSION THREAD

**Discussion by Warlord-K** [openvinotoolkit/openvino#16238] [Google Summer of Code]
Hi OpenVINO Developers!

I am Yatharth Gupta,CSE Undergrad at Indian Institute of Technology,Indore. I am really excited to contribute in the OpenVINO toolkit since I've used Neural Compressor in some of my previous projects to perform Pruning and Quantization, and used OpenVINO to optimize inference on CPU. Fresh Out of a Huge NLP Project and having previous experience in audio processing using torchaudio and librosa, I really want to contribute to the `Smart Meeting Notes` Project to be mentored by `Ekaterina Aidova` and `Andrei Kochin`. 

What is the preferred mode of communication with the mentors?

Also, I have finished the pre-requisite task mentioned in [Project Ideas](https://github.com/openvinotoolkit/openvino/wiki/Google-Summer-Of-Code#project-ideas-for-2023) and created my [Pull Request](https://github.com/openvinotoolkit/openvino_notebooks/pull/897). It would be really great if I could get an initial task to get a better understanding of the problem.

**Comment by adrianboguszewski** [comment]
@eaidova, please advice :)

**Reply by eaidova** [reply]
@Warlord-K thank you for your interest in our project1 Could you please share your opinion on how you see solution for the selected task?

**Reply by eaidova** [reply]
and yes, I'm here and ready to discuss if you have any questions :)

**Comment by Warlord-K** [comment]
Hi, I am really sorry, I forgot to check this discussion thread, since I was mainly working in OpenVINO Notebooks. I have submitted my Proposal with a detailed description of the following steps:

1) Speech Separation/ Speaker Diarization to separate the audio into different audio clips of the speakers.
2) Speech Recognition to convert the audio into text.
3) Text Summarization to convert the Transcript of the meet to the summary of the meet.
4) Translation of the Summary to various languages depending upon the requirement.

Other than these steps I will also incorporate speaker recognition using the Speaker Diarization data to get the photos of the speakers, and generate short video meeting highlights which would further aid the smart meeting notes.

@eaidova @andrei-kochin Please review my proposal and provide feedback.

**Comment by Warlord-K** [comment]
@eaidova @andrei-kochin Please review my proposal. I just had one major question that if i should add more detail or make the proposal more concise, and would love to get your feedback

**Reply by eaidova** [reply]
reviewed
